Source/santametricservice/Formats/SNTMetricMonarchJSONFormatTest.m

```diff
@@ -6,6 +6,17 @@
 @interface SNTMetricMonarchJSONFormatTest : XCTestCase
 @end
 
+// Stub out NSDate's date method
```

Comment:
It would be preferable to use OCMock for this, it's shorter and handles a lot of edge cases.

Reply:
Fair enough. We'd used this trick elsewhere. I'll move it to OCMock.